src/utils/github-data-parser.js

```diff
@@ -6,11 +8,18 @@
  *
  */
 export const parseUserData = (userData) => {
-  return userData.map(
-    ({ id, name, description, language, pushed_at, stargazers_count }) => {
-      return { id, name, description, language, pushed_at, stargazers_count };
-    },
-  );
+  return userData
+    .map(
+      ({
+        id,
+        name,
+        description,
+        language,
+        pushed_at: pushedAt,
+        stargazers_count: stargazersCount,
+      }) => ({ id, name, description, language, pushedAt, stargazersCount }),
+    )
+    .sort((a, b) => moment(b.pushedAt).diff(moment(a.pushedAt)));
```

Comment:
I dont think we should have the sorting logic here. Maybe on the user page we want the person to sort their repos alphabetically, by last updated, by last created and so on